packages/attention/src/component.tsx

```diff
@@ -70,15 +67,41 @@ export function Attention(props: AttentionProps) {
     },
   };
 
+  const activeAttentionProp = () => {
+    if (props.tooltip) {
+      return "tooltip";
+    } else if (props.callout) {
+      return "callout";
+    } else if (props.popover) {
+      return "popover";
+    } else {
+      return "";
+    }
+  }
+
+  const pointingAt = () => {
+    if (actualDirection === "bottom") {
+      return "up";
+    } else if (actualDirection === "top") {
+      return "down";
+    } else if (actualDirection === "right") {
+      return "left";
+    } else if (actualDirection === "left") {
+      return "right";
+    } else {
+      return "";
+    }
+  }
```

Comment:
`pointingAt` can in fact return `opposites[actualDirection]` and it will result in the same thing.

Reply:
Ah nice! :D